dockerplugin.db

```diff
@@ -8,3 +8,4 @@ network.usage           rx_bytes:COUNTER:0:U, rx_dropped:COUNTER:0:U, rx_errors:
 memory.usage            limit:GAUGE:0:U, max:GAUGE:0:U, total:GAUGE:0:U
 memory.stats            value:GAUGE:0:U
 memory.percent          value:GAUGE:0:150
+cpu                     value:GAUGE:0:U
```

Comment:
Hmm, I'd rather see more specific metric groups here for `cpu.quota` and `cpu.throttling_data`

Reply:
cpu.throttling actually exists. But you're right I can be more verbose by saying cpu.quota and cpu.shares